services/snyk/snyk-vulnerability-github.tester.js

```diff
@@ -0,0 +1,94 @@
+'use strict'
+
+const Joi = require('joi')
+const t = (module.exports = require('../create-service-tester')())
+const { colorScheme } = require('../test-helpers')
+const {
+  twoVulnerabilitiesSvg,
+  zeroVulnerabilitiesSvg,
+} = require('./snyk-test-helpers')
+
+t.create('live: valid repo')
+  .get('/badges/shields.json')
+  .expectJSONTypes(
+    Joi.object().keys({
+      name: 'vulnerabilities',
+      value: Joi.number().required(),
+    })
+  )
+
+t.create('live: non existent repo')
+  .get('/badges/not-real.json')
+  .timeout(10000)
```

Comment:
Asking Snyk for something that doesn't exist frequently takes a few seconds. In my local env this test would fail roughly every third/fourth test run so I added this timeout to avoid knowingly introducing a new, flaky test